# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/linux/hotfix-aha152x-printk-cont.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- linux-6.8/drivers/scsi/aha152x.c.vanilla	2024-05-07 00:51:23.119298807 +0200
+++ linux-6.8/drivers/scsi/aha152x.c	2024-05-07 00:57:34.281328460 +0200
@@ -833,7 +833,7 @@
 				"Please verify.\n", shpnt->host_no, shpnt->irq);
 		goto out_host_put;
 	}
-	printk("ok.\n");
+	printk(KERN_CONT "ok.\n");
 
 
 	/* clear interrupts */
